plugins/modules/iam_managed_policy.py

```diff
@@ -107,7 +107,7 @@
     state: absent
 '''
 
-RETURN = '''
+RETURN = r'''
 policy:
   description: Returns the policy json structure, when state == absent this will return the value of the removed policy.
   returned: success
```

Comment:
Not really sure here, but for me the returned policy is not a str but a dict instead. What do you think?

```
changed: [testhost] => {
    "changed": true,
    "invocation": {...},
    "policy": {
        "arn": "arn:aws:iam::721066863947:policy/ansible-test-policy",
        "attachment_count": 0,
        "create_date": "2022-02-04T15:02:07+00:00",
        "default_version_id": "v2",
        "is_attachable": true,
        "path": "/",
        "permissions_boundary_usage_count": 0,
        "policy_id": "AM......",
        "policy_name": "ansible-test-policy",
        "update_date": "2022-02-04T15:02:13+00:00"
    },
    "resource_actions": [
        "iam:CreatePolicyVersion",
        "iam:ListPolicies",
        "iam:ListPolicyVersions",
        "iam:GetPolicy",
        "iam:GetPolicyVersion",
        "iam:SetDefaultPolicyVersion"
    ]
}
```